components/clarity-repl/src/repl/remote_data/fs/node.rs

```diff
@@ -0,0 +1,39 @@
+use wasm_bindgen::prelude::*;
+
+use std::path::Path;
+
+#[wasm_bindgen(module = "fs")]
+extern "C" {
+    #[wasm_bindgen(js_name = readFileSync)]
+    fn read_file_sync(path: &str) -> Vec<u8>;
+    #[wasm_bindgen(js_name = existsSync)]
+    fn exists_sync(path: &str) -> bool;
+    #[wasm_bindgen(js_name = mkdirSync)]
+    fn mkdir_sync(path: &str, options: JsValue);
+    #[wasm_bindgen(js_name = writeFileSync)]
+    fn write_file_sync(path: &str, data: &[u8]);
+}
+
+pub fn get_file_from_cache(cache_location: &Path, name: &Path) -> Option<String> {
+    let cache_path = cache_location.join(name);
+    if !exists_sync(cache_path.to_str().unwrap()) {
+        None
+    } else {
+        Some(String::from_utf8(read_file_sync(cache_path.to_str().unwrap())).unwrap())
```

Comment:
The double unwrap might be fine. Is this just assumed to fail with a generic read_file_sync error?

Reply:
I reworked it with a few less unwrap, that was too mucn.

> Is this just assumed to fail with a generic read_file_sync error?

Not sure to understand the question.
But yes, i think it fine to simply panic and throw the fs error

Reply:
Cool 
sorry, this was meant to be non-blocking, but the change looks good 👍